docs/.vitepress/config.ts

```diff
@@ -779,83 +781,80 @@ export default defineConfig({
             },
           ],
         },
+
         {
-          text: '⛅️ Terramate Cloud 🚧',
-          collapsed: false,
+          text: '☁️ Terramate Cloud',
+          // link: '/cloud/',
           items: [
-            { text: 'Book a Demo', link: 'https://terramate.io/demo/' },
+            {
+              text: 'Book a demo',
```

Comment:
This section is out of scope and will create conflicts with other open PRs. pls revert.